xAPI.md

```diff
@@ -507,13 +507,39 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 7.0 xAPI Components
 
-## 8.0 Getting More Out of xAPI
 
-### 8.1 Extending xAPI
 
-### 8.2 Profiles
+## 8.0 Extending xAPI
+
+xAPI can be extended in a few ways.  The most notable and extensive is Statement Extensions, which allow great flexibility within Statements.  It is recommended that profiles or Communities or Practice agree on how to use  
```

Comment:
* double space
* delete "and extensive" as it sounds weird to call extensions extensive.